politeia_notifications.go

```diff
@@ -0,0 +1,57 @@
+package dcrlibwallet
+
+import (
+	"errors"
+)
+
+func (p *Politeia) AddNotificationListener(notificationListener ProposalNotificationListener, uniqueIdentifier string) error {
```

Comment:
All the contents in this file should go it politeia_sync.go, politeia files are getting too much.